nilearn/decoding/decoder.py

```diff
@@ -132,17 +133,69 @@ def _check_param_grid(estimator, X, y, param_grid=None):
         else:
             min_c = 0.5
 
-        if not isinstance(
-            estimator,
-            (RidgeCV, RidgeClassifierCV, DummyClassifier, DummyRegressor),
-        ):
+        if isinstance(estimator, (RidgeCV, RidgeClassifierCV)):
+            param_grid["alphas"] = [np.geomspace(1e-4, 100, num=20)]
+        elif not isinstance(estimator, (DummyClassifier, DummyRegressor)):
             param_grid["C"] = np.array([2, 20, 200]) * min_c
         else:
             param_grid = {}
 
+    else:
```

Comment:
to avoid a big block of code between this and the corresponding `if`, maybe we could have a `_default_param_grid(estimator)` helper function?